PageObjectPattern/src/test/java/serenity/pages/LoginPage.java

```diff
@@ -0,0 +1,40 @@
+package serenity.pages;
+
+import org.openqa.selenium.By;
+
+import net.serenitybdd.core.steps.UIInteractionSteps;
+import net.thucydides.core.annotations.Step;
+
+// Elements and Methods on Login Page
+
+public class LoginPage extends UIInteractionSteps {
+    MainPage mainPage;
+
+    // Elements located on Login page
+    By emailAddressField = By.name("email");
+    By passwordField = By.id("passwd");
+    By signInButton = By.id("SubmitLogin");
+    By forgotPasswordLink = By.xpath("//*[@title=\"Recover your forgotten password\"]");
+
+    // Methods used on login page
+    @Step("Open Login page")
+    public void openLoginPage() {
+        this.openUrl("http://automationpractice.com/index.php?controller=authentication&back=my-account");
+    }
+
+    @Step("User provides credentials")
```

Comment:
There is also a possibility to show values of the parameters on the report if you use for instance:
`User provides credentials - login {0}, password {1}`